### PR TITLE
Current ZAP swag

### PR DESCRIPTION
**Pull request by psiinon**
Copied directly from https://github.com/zaproxy/zap-swag

### CODE REVIEW

**Comment by psiinon** [comment]
I wont merge this myself, bad practice ;)

File: projects/ZAP/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+All ZAP images are released under the Creative Commons license.
+
+You can also purchase ZAP merchandise online - we make no money out of these sales, if you'd like to donate money to the OWASP ZAP project them please use the 'donate' button on https://www.owasp.org/index.php/ZAP
+
+You can of course use the artwork from this repo with any other online store that you like.
+
+## Stores
+A range of products can be purchased from http://www.redbubble.com/people/zaproxy
+
+Stickers can be purchased from https://www.stickermule.com/uk/user/1070684077/stickers
```
Comment (bkimminich) [inline projects/ZAP/README.md]: Stickermule user shops do not exist any more

Comment (psiinon) [reply projects/ZAP/README.md]: Oh yeah :/ I'll remove that...

**Comment by bkimminich** [comment]
I would only put those files in `owasp-swag` from which actual swag can reasonably be made. So, I would not add various sizes but only the largest available, definitely no tiny favicons or JPG variants etc.

What do you think?

File: projects/ZAP/README.md (hunk 1 above)
Comment (psiinon) [reply projects/ZAP/README.md]: Arg, and I left the ZAP dir in there too :(

**Comment by psiinon** [comment]
Yeah, that makes sense too

**Comment by psiinon** [comment]
Ha, didnt even get a chance to tell you I'd updated it ;)

**Pull request merged**